src/libs/asyncOpenURL/index.website.js

```diff
@@ -0,0 +1,29 @@
+import {Linking} from 'react-native';
+
+/**
+ * Prevents Safari from blocking pop-up window when opened within async call.
+ *
+ * @param {Promise} promise
+ */
+export default function asyncOpenURL(promise) {
```

Comment:
Also, what do you think about just providing the url as an argument to this function? Is that simpler/less prescriptive than this implementation, in which the promise needs to resolve to a url? Basically something more like [this implementation](https://expensify.slack.com/archives/C01GTK53T8Q/p1624062070455500?thread_ts=1623955103.415500&channel=C01GTK53T8Q&message_ts=1624062070.455500)

Reply:
Updated!